### PR TITLE
CoCo: enable annotations

### PR DESCRIPTION
**Pull request by fitzthum**
Update containerd config file to allow annotations for the CoCo runtime class.

This should unblock CI for https://github.com/kata-containers/kata-containers/pull/5665

@ryansavino 
@jimcadden

Fixes: #5513

### CODE REVIEW

**Review by fidencio — CHANGES_REQUESTED** [review]
@fitzthum, thanks for the contribution.

I've left a comment in-line, and I'd also suggest changing
```
Remove redundant containerd setup code from sev.bats and use the nice library code instead.
```

There's no need to use "the nice library code instead".  I'd reword it to "the already existent code instead".  It's as clear as yours, and folks can avoid reading it in a sarcastic way (which I'm pretty sure was **not** your intention).

File: integration/confidential/lib.sh [hunk 1]
```diff
@@ -203,6 +203,10 @@ configure_cc_containerd() {
 			"$containerd_conf_file"
 	fi
 
+	# Enable annotations for CoCo runtimeclass
+	sudo sed -i 's/\([[:blank:]]*\)\(cri_handler = "cc"\)/\1\2\n\1pod_annotations = ["io.katacontainers.\*"]/' \
+			"$containerd_conf_file"
```
[the remark this answers is not in the document]
Comment (fidencio) [reply integration/confidential/lib.sh]: My comments still stand with the new version of the code.

[the remark this answers is not in the document]
Comment (fitzthum) [reply integration/confidential/lib.sh]: Oh yeah, I much prefer adding it to this file as you suggest. I think there is a bit more thinking that we might want to do regarding differences between this config and the one we end up with from the operator.

**Review by fidencio — APPROVED** [review]
@fitzthum, lgtm, thanks!
